tests/server/collab/test_all.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+
+
+collab_model_names = ['projects', 'files', 'tasks', 'instances', 'vectors']
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize('model_name', collab_model_names)
+def test_empty_lists(client, model_name):
+  response = client.get('/collab/{}/'.format(model_name))
+  assert response.status_code is 200
+  json_response = response.json()
+  assert json_response == []
```

Comment:
This is meaningful - it means a client accessed those URLs and successfully received an empty list (opposed to an exception).

I'll add some more tests, adding random data and making sure its available.